tests/BinEnvJsonTest.php

```
@@ -0,0 +1,339 @@
+    private string $scriptPath;
+    private string $originalDir;
+    private string $tempDir;
+
+    protected function setUp(): void
+    {
+        $this->scriptPath = realpath(__DIR__ . '/../bin/envjson');
+        if ($this->scriptPath === false) {
+            $this->fail('bin/envjson script not found.');
+        }
+
+        $this->originalDir = getcwd();
```

PHPStan (2.0.3)

Property Koriym\EnvJson\BinEnvJsonTest::$originalDir (string) does not accept string|false.


(assign.propertyType)